fix incorrect multi-output format instructions (#1526)
semgrep/semgrep-docs: docs/getting-started/cli.md

semgrep ci --json --json-output=semgrep.json
```

In addition to the `--text`, `--json`, and `--sarif` flags, which set the primary output formats, and the `--output=<value>` flag that saves the results to a file or posts to a URL, you can append `--<format>-output=<file>` to obtain additional output streams:

```console
# prints findings in SARIF format to standard output and writes in JSON format to `findings.json`.
semgrep ci --sarif --json-output=findings.json

> # prints findings in text to standard out and JSON output to `findings.json`.

Reviewer: ```suggestion
# prints findings in text to standard out and writes JSON output to `findings.json`.
```

semgrep ci --json-output=findings.json

# prints text output to `findings.txt` and writes in SARIF to `findings.sarif`.
semgrep ci --output=findings.txt --sarif-output=findings.sarif

> # prints text to standard output and saves to JSON and SARIF files

Reviewer: ```suggestion
# writes text to `semgrep.txt`, JSON to `semgrep.json`, and SARIF to `semgrep.sarif`.
```

Reviewer: Note: standard output here is wrong. Maybe it was wrong in my example?

```console
# obtain results as text output and as JSON, SARIF

> semgrep ci --text --text-output=semgrep.txt --json-output=semgrep.json --sarif-output=semgrep.sarif

Reviewer: ```suggestion
semgrep ci --text --output=semgrep.txt --json-output=semgrep.json --sarif-output=semgrep.sarif
```
